serverless/DEVELOPER_README.md

```diff
@@ -280,7 +280,11 @@ The user **deploying** this project **must** have the following permissions list
 				"apigateway:PUT",
 				"apigateway:TagResource"
 			],
-			"Resource": "*"
+			"Resource": [
+                "arn:aws:apigateway:<region>::/apis/*",
+                "arn:aws:apigateway:<region>::/apis",
```

Comment:
Do we require all of these resources?

Reply:
Using just `"arn:aws:apigateway:<region>::/apis/*"` was not enough, these other resources were necessary.